### PR TITLE
[SWUI-23][IMPROV] - Monorepo - turn the 2 projects into one

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWUI-23](https://linear.app/swaprhq/issue/SWUI-23/monorepo-turn-the-2-projects-into-one)

# Description
* Create a monorepo for Swapr-UI using Turbo
* Fix renamed imports
* Remove build warning for unwanted `"use client"` directive

### CODE REVIEW

**Review by berteotti — COMMENTED** [review]
Let's make the 2 packages inside the same folder `/packages`. And let's call the UI `/swapr-ui` and the other `/swapr-ui-website`

File: package.json [hunk 1]
```diff
@@ -1,84 +1,29 @@
 {
   "name": "swapr-ui",
```
Comment (berteotti) [inline package.json:2]: Let's update this name

File: packages/components/package.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@swapr-ui/components",
```
Comment (berteotti) [inline packages/components/package.json:2]: Let's call this `swapr-ui` for now

File: apps/website/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@swapr-ui/website",
```
Comment (berteotti) [inline apps/website/package.json:2]: Let's call this `swapr-ui-website` for now

File: package.json (hunk 1 above)
Comment (berteotti) [reply package.json:2]: After checking some examples, lets remove the name

**Comment by ElRodrigote** [comment]
> Let's make the 2 packages inside the same folder `/packages`. And let's call the UI `/swapr-ui` and the other `/swapr-ui-website`

As we spoke off-line, we will keep the `packages` and the `apps` folders as our Website is not a package, but will proceed with the repositories renaming

**Comment by Diogomartf** [comment]
Let's build with the failing build on another PR.

**Review by Diogomartf — APPROVED** [review]

**Review by berteotti — APPROVED** [review]

**Pull request merged**